testing/kuttl/e2e-other/exporter-upgrade/00-assert.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: exporter
+status:
+  instances:
+    - name: instance1
+      readyReplicas: 1
+      replicas: 1
+      updatedReplicas: 1
```

Comment:
Should we check that the exporter status is set? Maybe we should be doing this in the `e2e` test :thinking:

Reply:
I think we run into the problem we've had with KUTTL where checking anything in an array becomes tricky if we only care about one thing. (Like the `exporter` entry in the `containerStatuses`.) The 01--check-exporter.yaml would fail if the exporter wasn't working, and at least in the `e2e`, that exporter will be working by the time the backup is finished.

(Here, because I'm using an outdated image, the backup isn't finishing, which is why I pulled that part out of 00-assert and why I put a retry in 001-check-exporter.)